js/index.js

```diff
@@ -0,0 +1,26 @@
+$(document).ready(function() {
+
+	$('.read-more').click(showMore);
+	$('.read-less').click(showLess);
+
+	function showMore() {
+		$('#readMoreText').slideDown();
+		$('.read-more').hide();
+		$('.read-less').show();
+	}
+
+	function showLess() {
+		$('#readMoreText').slideUp(function() {
+			$('.read-less').hide();
+			$('.read-more').show();
```

Comment:
Nice! Only running the `hide` and `show` when the `.slideUp()` animation has finished.